CONTRIBUTING.rst

```diff
@@ -175,131 +175,3 @@ Note: The build requires a lot of different versions of python, so rather than h
 the build system will install them itself in a local directory. This means that the first time you run a task you
 may have to wait a while as the build downloads and installs the right version of python for you.
 
```

Comment:
Looks like the linter wants you to delete this line too, then we should be ready to merge!